examples/nextjs-sceduler/.env.production

```diff
@@ -0,0 +1,2 @@
+NEXT_PUBLIC_YORKIE_API_ADDR='https://api.yorkie.dev'
+NEXT_PUBLIC_YORKIE_API_KEY='cedaovjuioqlk4pjqn6g'
```

Comment:
Same for here.

Reply:
I just fixed it. thank you for your comments :)